dev/com.ibm.ws.jsonb_fat/test-applications/jsonbapp/src/web/jsonbtest/JohnzonTestServlet.java

```diff
@@ -10,35 +10,41 @@
  *******************************************************************************/
 package web.jsonbtest;
 
+import static componenttest.annotation.SkipForRepeat.EE9_FEATURES;
 import static web.jsonbtest.JSONBTestServlet.PROVIDER_JOHNZON;
 import static web.jsonbtest.JSONBTestServlet.PROVIDER_YASSON;
 
 import javax.servlet.annotation.WebServlet;
 
 import org.junit.Test;
 
+import componenttest.annotation.SkipForRepeat;
 import componenttest.app.FATServlet;
 
 @SuppressWarnings("serial")
 @WebServlet(urlPatterns = "/JohnzonTestServlet")
 public class JohnzonTestServlet extends FATServlet {
 
     @Test
+    @SkipForRepeat(EE9_FEATURES)
```

Comment:
can you include a comment indicating why these tests are skipped?

Reply:
done, thanks!